### PR TITLE
Next 14

### PR DESCRIPTION
**Pull request by AndreMaz**
This PR updates Moleculer to v0.14-beta7.
Changes:
- updates all deps to latest versions
- `blog` - updated docker files to work with node 12
- `conduit` - removed additional logic to make the example compliant with [RealWorld API Spec](https://github.com/gothinkster/realworld/tree/master/api)
- Drop Promises and use async/await

Some prints:
**Blog + Docker Microservices**
image
**Blog + Docker Monolith**
image

**Conduit + Docker + Postman API Tests**
image

### CODE REVIEW

File: blog/Dockerfile [hunk 1]
```diff
@@ -1,30 +1,14 @@
...
+WORKDIR /app
 
-ENV HOME=/home/mol
-ENV HOME_APP=$HOME/app
-ENV NODE_ENV=production 
-
-WORKDIR $HOME_APP
+ENV NODE_ENV=production
 
 COPY package.json .
```
Comment (icebob) [inline blog/Dockerfile]: Please use `COPY package*.json ./`

File: blog/package.json [hunk 2]
```diff
@@ -11,30 +11,31 @@
...
-    "moleculer-fake": "1.0.11",
-    "moleculer-repl": "0.5.0",
-    "moleculer-web": "0.8.0",
-    "moment": "2.22.2",
-    "mongoose": "5.2.1",
-    "morgan": "1.9.0",
-    "nats": "1.0.0",
-    "pug": "2.0.3",
-    "slugify": "1.3.0"
+    "bluebird": "^3.7.2",
```
Comment (icebob) [inline blog/package.json]: Bluebird is used in the project?

File: conduit/services/articles.service.js [hunk 3]
```diff
@@ -187,22 +187,6 @@ module.exports = {
 				if (ctx.params.tag)
 					params.query.tagList = { "$in" : [ctx.params.tag] };
 
-				if (ctx.params.author) {
```
Comment (icebob) [inline conduit/services/articles.service.js:190]: Please put back these parts just commenting out.

File: blog/docker/mixed/docker-compose.yml [hunk 4]
```diff
@@ -44,20 +44,20 @@ services:
...
-    environment:
-      NODEID: "node-metrics"
-      SERVICES: metrics
-    links:
-      - nats
-      - redis
-    depends_on:
-      - nats
-      - redis
+  #metrics:
```
Comment (icebob) [inline blog/docker/mixed/docker-compose.yml]: You can remove it.

File: blog/docker/micro/docker-compose.yml [hunk 5]
```diff
@@ -95,20 +95,20 @@ services:
...
-    environment:
-      NODEID: "node-metrics"
-      SERVICES: metrics
-    links:
-      - nats
-      - redis
-    depends_on:
-      - nats
-      - redis
+  #  metrics:
```
Comment (icebob) [inline blog/docker/micro/docker-compose.yml]: You can remove it.

**Review by icebob — CHANGES_REQUESTED** [review]

File: blog/docker/micro/docker-compose.yml (hunk 5 above)
Comment (AndreMaz) [reply blog/docker/micro/docker-compose.yml]: Done in https://github.com/moleculerjs/moleculer-examples/pull/22/commits/d226aa09a9302e9dd27e55c4f6291aa5fc555a22

File: blog/docker/mixed/docker-compose.yml (hunk 4 above)
Comment (AndreMaz) [reply blog/docker/mixed/docker-compose.yml]: Done in https://github.com/moleculerjs/moleculer-examples/pull/22/commits/d226aa09a9302e9dd27e55c4f6291aa5fc555a22

File: conduit/services/articles.service.js (hunk 3 above)
Comment (AndreMaz) [reply conduit/services/articles.service.js:190]: Done in https://github.com/moleculerjs/moleculer-examples/pull/22/commits/d226aa09a9302e9dd27e55c4f6291aa5fc555a22

File: blog/package.json (hunk 2 above)
Comment (AndreMaz) [reply blog/package.json]: No, I've removed it in https://github.com/moleculerjs/moleculer-examples/pull/22/commits/d226aa09a9302e9dd27e55c4f6291aa5fc555a22

File: blog/Dockerfile (hunk 1 above)
Comment (AndreMaz) [reply blog/Dockerfile]: Done in https://github.com/moleculerjs/moleculer-examples/pull/22/commits/d226aa09a9302e9dd27e55c4f6291aa5fc555a22
I've also made the same change in `conduit/Dockerfile`

**Review by icebob — APPROVED** [review]
Great job, thank you!

**Pull request merged**